lib/matplotlib/cm.py

```diff
@@ -16,11 +16,11 @@
 import matplotlib as mpl
 import matplotlib.colors as colors
 import matplotlib.cbook as cbook
-from matplotlib._cm import datad
+from matplotlib._cm import datad, _deprecation_datad
 from matplotlib._cm import cubehelix
 from matplotlib._cm_listed import cmaps as cmaps_listed
 
-cmap_d = dict()
+cmap_d = _deprecation_datad()
```

Comment:
Note that line 88 of this file will likely trigger the deprecation warning if I'm not wrong, as it loop through all the colormap to build the reversed ones. Not sure what we can do about the though. Just pointing it out.